cohttp/src/jbuild

```diff
@@ -2,7 +2,7 @@
 #require "unix"
 
 let version =
-  let ic = open_in "../VERSION" in
+  let ic = open_in "../../VERSION" in
```

Comment:
@hhugo @diml -- is it worth an upstream jbuilder issue to track how to improve creating version file stamps?  probably good not to to propagate too many hacky rules throughout the ecosystem.